net/unbound/files/unbound.sh

```diff
@@ -1047,9 +1026,28 @@ unbound_uci() {
 
 ##############################################################################
 
+unbound_ismain() # true=0, false=1.
+{
+  if [ "$UNBOUND_N_RX_PORT" != "53" ] ; then
```

Comment:
Couldn't DNAT port rewrites (atomic) help in a back-to-back testing setup ?
(Hans Dedecker questioned 53 as a condition for main dnsmasq instance in the light of possible port rewrites).

Re, is the re-ordering of the tests problematic or does the port-conflict-test conflict with your back-to-back testing ?

ad 1 and 4) Would unbound for this reason not listen on a non-53 port by default ?
ad 4) ftp/samba hack scripts seem hardly a practical use-case to cover.